src/workflows/calls_generation.py

```diff
@@ -52,8 +52,8 @@ def pipeline(
                 "model_name": generation_model,
                 "language": language,
                 "fields": [
-                    "first_name: in spanish, no special characters",
-                    "last_name: in spanish, no special characters",
+                    f"first_name: in {language}, no special characters",
+                    f"last_name: in {language}, no special characters",
                     "phone_number",
                     "email",
                     "client_id",
```

Comment:
If already improved this, it's better to make sure also there are no leading zeros in the id (will fail the function)
```suggestion
                    "client_id, no leading zeros",
```